pydantic_redis/abstract.py

```diff
@@ -32,7 +35,7 @@ class _AbstractModel(BaseModel):
     @classmethod
     def serialize_partially(cls, data: Optional[Dict[str, Any]]) -> Dict[str, Any]:
         """Converts non primitive data types into str"""
-        return {key: orjson.dumps(value) for key, value in data.items()}
+        return {key: orjson.dumps(value, default=pydantic_serializer) for key, value in data.items()}
 
     @classmethod
     def deserialize_partially(cls, data: Optional[Dict[bytes, Any]]) -> Dict[str, Any]:
```

Comment:
Let's also add a test case for such a model that has attributes with annotations like `List[Model]` (and probably Dict[Any, Model], Tuple[Model]) in [test_pydantic_redis.py](../test/test_pydantic_redis.py)